jenkins/prepare-yum-repos

```diff
@@ -27,28 +27,20 @@ STORAGE='http://cortx-storage.colo.seagate.com'
 
 cat <<EOF > /etc/yum.repos.d/motr_last_successful.repo
 [motr-dev]
-baseurl=$STORAGE/releases/cortx/github/main/centos-7.9.2009/last_successful/
+baseurl=$STORAGE/releases/cortx/github/main/rockylinux-8.4/last_successful/
```

Comment:
One concern though, if tomorrow we need to use some other repo we will again have to change this right, is there a way to somehow pass the repo as a parameter to this script?

Reply:
@pavankrishnat any change needed for this comment from @mssawant ? or we are good to merge for now?

Reply:
@mssawant Changed Jenkins Groovy script as requested, now we can send repo path from Jenkins.